test/e2e/storage/vsphere/bootstrap.go

```diff
@@ -35,10 +38,19 @@ func Bootstrap() {
 }
 
 func bootstrapOnce() {
-	// TBD
 	// 1. Read vSphere conf and get VSphere instances
-	// 2. Get Node to VSphere mapping
-	// 3. Set NodeMapper in vSphere context
-	TestContext = Context{}
+	vsphereInstances, err := GetVSphereInstances()
+	if err != nil {
+		framework.Failf("Failed to bootstrap vSphere with error: %v", err)
+	}
+	// 2. Get all ready nodes
+	nodeList := framework.GetReadySchedulableNodesOrDie(f.ClientSet)
+	TestContext = VSphereContext{NodeMapper: &NodeMapper{}}
+
+	// 3. Get Node to VSphere mapping
+	err = TestContext.NodeMapper.GenerateNodeMap(vsphereInstances, *nodeList)
+	if err != nil {
+		framework.Failf("Failed to bootstrap vSphere with error: %v", err)
+	}
```

Comment:
If NodeMap is successfully populated, can we log its content here once before returning from `bootstrapOnce`. It would be very helpful while debugging.